index.js

```diff
@@ -76,13 +77,19 @@ class LiveStrategyExecution extends EventEmitter {
 
     const { includeTrades, symbol, tf } = this.strategyOpts
     const candleKey = `trade:${tf}:${symbol}`
+    let lastUpdate = 0
```

Comment:
we should separate the `lastUpdate` time for candle and trade .. because, the `trade.mts` will always be greater than `candle.mts` in case of receiving the data for updated candle and will always ignore the candle update data.

Reply:
candles are composed of trades, so if the user chooses to include trades that will be the best/fast option for me to have a price update and track performance.